src/components/TranslationLeaderboard.js

```diff
@@ -0,0 +1,245 @@
+// Libraries
+import React, { useState } from "react"
+import styled from "styled-components"
+
+// Components
+import Emoji from "./Emoji"
+import { ButtonSecondary } from "./SharedStyledComponents"
+import Translation from "./Translation"
+
+// Data
+import monthData from "../data/translation_reports/month_data.json"
+import quarterData from "../data/translation_reports/quarter_data.json"
+import allTimeData from "../data/translation_reports/alltime_data.json"
+
+const Content = styled.div`
+  margin-bottom: 2rem;
+`
+
+const Table = styled.div`
+  background-color: ${(props) => props.theme.colors.background};
+  box-shadow: ${(props) => props.theme.colors.tableBoxShadow};
+  width: 100%;
+  margin-bottom: 2rem;
+`
+
+const Header = styled.div`
+  background-color: ${(props) => props.theme.colors.grayBackground};
+  text-decoration: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: ${(props) => props.theme.colors.text} !important;
+  margin-bottom: 1px;
+  padding: 1rem;
+  width: 100%;
+  color: #000;
+`
+
+const Item = styled.div`
+  text-decoration: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: ${(props) => props.theme.colors.text} !important;
+  box-shadow: 0 1px 1px ${(props) => props.theme.colors.tableItemBoxShadow};
+  margin-bottom: 1px;
+  padding: 0.5rem 1rem;
+  width: 100%;
+  color: #000;
+  &:hover {
+    border-radius: 4px;
+    box-shadow: 0 0 1px ${(props) => props.theme.colors.primary};
+    background: ${(props) => props.theme.colors.tableBackgroundHover};
+  }
+`
+
+const TextContainer = styled.div`
+  flex: 1 1 75%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-right: 2rem;
+  overflow-wrap: anywhere;
+`
+
+const WordsContainer = styled.div`
+  flex: 1 1 36%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`
+
+const Avatar = styled.img`
+  margin-right: 1rem;
+  height: 20px;
+  width: 20px;
+  border-radius: 50%;
+  @media (max-width: ${(props) => props.theme.breakpoints.xs}) {
+    display: none;
+  }
+`
+
+const ItemNumber = styled.div`
+  margin-right: 1rem;
+  opacity: 0.4;
+`
+
+const ButtonRow = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 1rem;
+  flex-wrap: wrap;
+`
+
+const ButtonSecondaryStyled = styled(ButtonSecondary)`
```

Comment:
I like the look of these buttons but they seem too close to the table & overall the spacing feels tight:
![Image 2021-11-29 at 7 56 09 PM](https://user-images.githubusercontent.com/8097623/143971845-f91361bd-b753-4bf2-ae52-76dc37d5853a.jpg)

I feel like we could work off existing elements, e.g. the pills of the dapps page:
![Image 2021-11-29 at 7 55 54 PM](https://user-images.githubusercontent.com/8097623/143971896-b01af833-c397-4193-b47a-6bb7b7858816.jpg)

Reply:
Really whenever possible I'd encourage us to re-use similar elements for consistency (& ideally sharing the styles in our code).